dotcom-rendering/src/web/components/TopicFilterBank.importable.tsx

```diff
@@ -96,7 +96,7 @@ export const TopicFilterBank = ({
 	return (
 		<div css={containerStyles}>
 			<div css={headlineStyles}>
-				Filters <span css={headlineAccentStyles}>(BETA)</span>
+				Filters <span css={headlineAccentStyles}>(BETA):</span>
```

Comment:
Are the styles for the BETA text meant to be applied to the colon too?

Reply:
Oh you're right @AshCorr I shouldv'e considered this. Will quickly fix that now